eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -254,6 +254,52 @@
           "old": "method org.springframework.boot.autoconfigure.kafka.KafkaProperties com.azure.spring.cloud.autoconfigure.eventhubs.kafka.AzureEventHubsKafkaAutoConfiguration::azureKafkaProperties(com.azure.spring.cloud.core.provider.connectionstring.ServiceConnectionStringProvider<com.azure.spring.cloud.core.service.AzureServiceType.EventHubs>)",
           "justification": "To move kafka properties customization to bean post processor."
         },
+        {
+          "code": "java.annotation.attributeValueChanged",
+          "old": "class com.azure.spring.cloud.autoconfigure.aad.AadAutoConfiguration",
+          "new": "class com.azure.spring.cloud.autoconfigure.aad.AadAutoConfiguration",
```

Comment:
Is "new" necessary here?

Reply:
Is it possible to "To support JWT Client Authentication" without this change?

Reply:
Same to all other changes in current file. 
If breaking change is necessary, please add more description in "justification".